src/sacn/private/opts.h

```diff
@@ -150,6 +150,17 @@
  * @{
  */
 
+/*!
+ * \brief Whether or not the receiver will be supporting IPv6.
+ *
+ * Turns on/off IPv6 support on the receiver.
+ *
+ * //CHRISTIAN TODO: Add IPV6 support, and we should talk about if this should default to 1...
```

Comment:
I don't think this option is necessary, IPv6 support should always be enabled on systems that support it.

Reply:
I guess my thinking was that this option would be how we know they support it.  Just because an underlying platform supports IPv6, doesn't mean we want the device to support it.  There's extra configuration & UI involved, for example.

But I'm open to removing this if you feel strongly.

Reply:
Because of the way we enumerate network interfaces, libraries like sACN will only use IPv6 if there is an IPv6-enabled network interface on the system. Devices which don't wish to support IPv6 will have it disabled in the network stack, and therefore no network interfaces will show up with IPv6 addresses. Socket calls with AF_INET6 might also fail, which would be able to be handled at runtime. If a device DOES have IPv6 support enabled, it will generate IPv6 addresses for its network interfaces, so if those are not configurable or accessible from the UI, that should be considered to be a device bug, not an sACN library bug.

Reply:
A very good point!  I'll remove the option.